### DISCUSSION TITLE
Is there a driver for Linux Servers?

### DISCUSSION THREAD

**Discussion by codingnetworksb** [scrapli/scrapli_community#179] [Q&A] [answered]
I didn't find a Linux server driver in the list. Before creating one and submit a PR, i want to ask if there is one that works well with any linux system like ubuntu, redhat, etc.

I'm working on tasks automations on a variety of devices. I don't want to use a different library to automate tasks with Linux servers.

**Comment by carlmontanari** [comment, marked as answer]
hey @codingnetworksb -- generic driver should be fine for nix things. worst case you may have to make the prompt pattern a bit more lax to catch things (esp if you have fancy prompts or whatever). that said in general id say unless its just some little thing id say there are probably better tools for nix things than scrapli (ansible/puppet and the like, or maybe something like fabric for example).

all that to say -- I likely wouldn't merge a pr for linux-y stuff since id just say use generic, so hopefully that saves ya some time! :D
